### PR TITLE
[ADD] Migrated sale_cancel_reason module in v11.

### PR DESCRIPTION
**Pull request by sudhir-serpentcs**
Migrated sale_cancel_reason module in v11.

### CODE REVIEW

**Comment by js-landoo** [comment]
Hi @sudhir-serpentcs 

I'm interested in the migration of this module. I see you have it almost done but it's stopped 7 months ago.
Are you planning to finish it?
Can I help you in any way?

Thanks and best regards.

**Comment by JayVora-SerpentCS** [comment]
@js-landoo Please proceed to make your PR.

**Comment by js-landoo** [comment]
OK @JayVora-SerpentCS we will proceed. Thanks.

**Comment by JayVora-SerpentCS** [comment]
I assume the history is not preserved, so we must close this PR. @pedrobaeza .
@Swapnesh-SerpentCS  can make new one.

File: sale_cancel_reason/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 Guewen Baconnier, Camptocamp SA
```
Comment (nikul-serpentcs) [inline sale_cancel_reason/__init__.py:2]: Remove Copyright From all `__init__.py` file.

File: sale_cancel_reason/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 Guewen Baconnier, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Sale Cancel Reason',
+    'version': '11.0.1.0.0',
+    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'category': 'Sale',
+    'license': 'AGPL-3',
+    'website': "http://www.camptocamp.com",
```
Comment (nikul-serpentcs) [inline sale_cancel_reason/__manifest__.py:10]: Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

File: sale_cancel_reason/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    'depends': [
+        'sale',
+    ],
+    'data': [
+        'wizard/cancel_reason_view.xml',
```
Comment (nikul-serpentcs) [inline sale_cancel_reason/__manifest__.py:15]: Fix sequence also

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by nikul-serpentcs** [comment]
If Create new PR then adding all comments & preserv history

**Comment by rafaelbn** [comment]
Closing as indicated by @JayVora-SerpentCS in https://github.com/OCA/sale-workflow/pull/559#issuecomment-425939512

**Pull request closed without merging**